man/systemd.preset.xml

```diff
@@ -84,6 +84,9 @@
     <para>If multiple lines apply to a unit name, the first matching
     one takes precedence over all others.</para>
 
+    <para>Values for unit templates should be specified as a space separated
+    list at the end of the line instead of customarily after the @.</para>
```

Comment:
Hmm, this para seems to come out completely out of the blue. Are you trying to say that 'enable asdf@.service foo' should be used in preference to 'enable asdf@foo.service'?

I'd expect a sentence about this syntax in the first paragraph above after "... seperated by newlines.":
> For template units, the unit name may be followed by a list of instances.
> If the list of instances is specified, those instances will be enabled (see example below).
> Otherwise, the default instance will be enabled (see DefaultInstance= in systemd.unit(5)).

Reply:
> Hmm, this para seems to come out completely out of the blue. Are you trying to say that 'enable [asdf@.service](mailto:asdf@.service) foo' should be used in preference to 'enable [asdf@foo.service](mailto:asdf@foo.service)'?

I now read #15065, and I see what this is about. Hopefully the bug will be fixed soon, so there's no reason to mention this issue in the documentation.

Reply:
Even with the bug fixed, I think it would be nice to document the syntax somewhere, but with the bug fixed it should definitely be a "may" instead of a "should".

Reply:
I agree that the syntax should be described, something like what I suggested above. The part that is *not* necessary is recommending one syntax over the other.